tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_device_RTL.py

```diff
@@ -36,27 +58,36 @@ def test_settings_app(self):
         device_info_page.exit_update_frequency()
         settings.return_to_prev_menu(settings.screen_element)
 
-        ### Downloads page
+        ###################### Download and its sub pages ######################
         settings.open_downloads()
         self.take_screenshot('downloads')
         settings.return_to_prev_menu(settings.screen_element)
-
-        settings.open_battery()
+        ###################### Battery and its sub pages ######################
+        battery_page = settings.open_battery()
         self.take_screenshot('battery')
+        battery_page.tap_turn_on_auto()
+        self.take_screenshot('battery-turnon_options')
+        battery_page.select('never')
         settings.return_to_prev_menu(settings.screen_element)
 
-        settings.open_accessibility()
+        ###################### Accessibility and its sub pages ######################
+        access_page = settings.open_accessibility()
         self.take_screenshot('accessibility')
+        color_page = access_page.a11y_open_color_settings()
+        color_page.a11y_toggle_filters()
```

Comment:
We should not be using accessibility methods in our Gaia UI tests and imagecompare tests, I believe. They need to be separate. We're not supposed to maintain/fix problems in the accessibility suite.